spec/components/errormessageoverlay.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { UIInstanceManager } from './../../src/ts/uimanager';
+import { ErrorMessageOverlay } from '../../src/ts/components/errormessageoverlay';
+import { MobileV3PlayerEvent } from '../../src/ts/mobilev3playerapi';
+import { MockHelper, TestingPlayerAPI } from '../helper/MockHelper';
+
+describe('ErrorMessageOverlay', () => {
+  describe('configure', () => {
+    let errorMessageOverlay: ErrorMessageOverlay;
+    let playerMock: TestingPlayerAPI;
+    let uiInstanceManagerMock: UIInstanceManager;
+  
+    beforeEach(() => {
+      errorMessageOverlay = new ErrorMessageOverlay({});
+      playerMock = MockHelper.getPlayerMock();
+      uiInstanceManagerMock = MockHelper.getUiInstanceManagerMock();
+    });
+  
+    it('should add event listener for source loaded event', () => {
```

Comment:
I know we have differences in using the `should` word in test names 😄 (Up to you if you change that - No blocker still would prefer not using it) **all occurrences**

```suggestion
    it('adds an event listener for source loaded event', () => {
```